packages/@aws-cdk/cli-lib/THIRD_PARTY_LICENSES

```diff
@@ -1,6 +1,6 @@
 The @aws-cdk/cli-lib package includes the following third-party software/licensing:
 
-** @jsii/check-node@1.75.0 - https://www.npmjs.com/package/@jsii/check-node/v/1.75.0 | Apache-2.0
+** @jsii/check-node@1.76.0 - https://www.npmjs.com/package/@jsii/check-node/v/1.76.0 | Apache-2.0
```

Comment:
Pretty sure this is updated by our automation, right?

Reply:
I think the automation actually misses this one. I need to fix that.